Formula/t/tmux.rb

```diff
@@ -92,6 +92,8 @@ def caveats
     <<~EOS
       Example configuration has been installed to:
         #{opt_pkgshare}
+      You might need to kill existing tmux servers as a result of this upgrade:
+        'tmux kill-server'
```

Comment:
That's fair. I thought it might be worth noting simply as a caveat of tmux. I hadn't realized the caveats section was for homebrew-specific notes.

I still think there's value to presenting this info. What is the best place, if not the caveats section?

Reply:
> I hadn't realized the caveats section was for homebrew-specific notes.

Perhaps you might have been confused by the existence of some very old caveats that are not Homebrew-specific. We avoid touching those unless we know they're wrong, but, for new caveats we require that they be Homebrew-specific.

We have this requirement for two reasons:
1. We want to avoid the caveats becoming stale, which is a big risk when the caveat is non-specific.
2. Non-specific caveats are useful to non-Homebrew users too, so they're better placed in upstream documentation. (They're also less likely to go stale there.)

> I still think there's value to presenting this info. What is the best place, if not the caveats section?

Given 2 above, I suggest trying to add this to the tmux `README`/FAQ/etc.

Reply:
Yeah, I can see why you'd want to avoid having external caveats. Thank you for clarifying!